lib/bolt/module_installer/puppetfile/module.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'bolt/error'
+
+module Bolt
+  class ModuleInstaller
+    class Puppetfile
+      class Module
+        attr_reader :full_name, :name, :type
+
+        def initialize(name)
+          @full_name, @name = parse_name(name)
+        end
+
+        # Formats the full name and extracts the module name.
+        #
+        protected def parse_name(name)
```

Comment:
I don't understand why this is different from the one in forge_module?

Reply:
Just forgot to remove it from forge_module...